src/bin/zmimapdctl

```diff
@@ -0,0 +1,128 @@
+#!/bin/bash
```

Comment:
Can we run this through shellcheck (https://www.shellcheck.net/).  I'd like to see us use that as a linter for an bash scripts we use in the future.  It'll give you a few suggestions on using $() over ``, and unused variable or two, and suggestions on quoting in a few places too.